iot-hub/Samples/service/FileUploadNotificationReceiverSample/FileUploadNotificationReceiverSample.cs

```diff
@@ -79,13 +78,7 @@ private async Task ReceiveFileUploadNotifications(string targetDeviceId, Cancell
             {
                 try
                 {
-                    FileNotification fileUploadNotification = await notificationReceiver.ReceiveAsync(s_notificationReceiverTimeout);
-
-                    if (fileUploadNotification == null)
```

Comment:
How about keep the null check here and just update the logging message to not include the field `s_notificationReceiverTimeout`